API.md

```diff
@@ -356,23 +356,23 @@
 
   Examples:
 
-  * If compiled with `FBINK_FOR_KINDLE`:
+  * For Kindle:
 
     ```sh
     fbink -t regular=/usr/java/lib/fonts/Caecilia_LT_65_Medium.ttf,bold=/usr/java/lib/fonts/Caecilia_LT_75_Bold.ttf,size=24,top=100,bottom=500,left=25,right=50,format "Hello **world**!"
     ```
 
     Will use Caecilia to print "Hello world!" at 24pt in a display area starting from 100px down the top of the screen to 500px before the bottom of the screen, from 25px of the left edge of the screen until 50px before the right edge. Honoring the Markdown syntax, "Hello" will be printed with the Regular font style, while "world" will use the Bold font style. You will NOT be able to use obfuscated or encrypted fonts.
 
-  * If not compiled with `FBINK_FOR_KINDLE`:
+  * For other than Kindle:
```

Comment:
The only thing that differs here is the path being used in the example, it's not really tied to hardware/platform quirks, so I'd possibly amend those mentions to `on` Kindle/Kobo.

(Or any other wording you may find that conveys the fact that this is just an example using paths specific to *a* specific platform, namely Kindle and then Kobo).

Reply:
How about 3e7ce40a7b1b1c8dcfa10a45af11defeabdf59ee?